packages/ckeditor5-image/tests/imagestyle/imagestyleui.js

```diff
@@ -248,6 +248,9 @@ describe( 'ImageStyleUI', () => {
 				expect( buttonView.tooltip ).to.be.true;
 				expect( buttonView.class ).to.be.null;
 
+				expect( buttonView.arrowView.label ).to.equal( ( config.title ? `${ config.title }: ` : '' ) + defaultItem.title );
```

Comment:
The code \`( config.title ? ``${ config.title }: ` : '' ) + defaultItem.title`` was confusing on its own, now that it's duplicated it's really asking for extracting to a reasonable named variable to be reused 🙂